### PR TITLE
[opentelemetry-operator] - bump operator/collector images

### PR DESCRIPTION
**Pull request by Allex1**

### CODE REVIEW

File: charts/opentelemetry-operator/Chart.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: dmitryax
   - name: TylerHelmuth
 icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e9e3ac0c8817cb/projects/opentelemetry/icon/color/opentelemetry-icon-color.png
-appVersion: 0.56.0
+appVersion: 0.58.0
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/Chart.yaml:14]: Should this be 0.59.0?

File: charts/opentelemetry-operator/values.yaml [hunk 2]
```diff
@@ -16,10 +16,10 @@ imagePullSecrets: []
 manager:
   image:
     repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    tag: v0.56.0
+    tag: v0.58.0
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/values.yaml:19]: Should this be 0.59.0?

File: charts/opentelemetry-operator/Chart.yaml (hunk 1 above)
Comment (Allex1) [reply charts/opentelemetry-operator/Chart.yaml:14]: nope https://github.com/open-telemetry/opentelemetry-operator/releases

File: charts/opentelemetry-operator/values.yaml (hunk 2 above)
Comment (Allex1) [reply charts/opentelemetry-operator/values.yaml:19]: nope https://github.com/open-telemetry/opentelemetry-operator/releases

File: charts/opentelemetry-operator/Chart.yaml (hunk 1 above)
Comment (Allex1) [reply charts/opentelemetry-operator/Chart.yaml:14]: [This](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/352/files#diff-56c52957e9b80d7dcb9b9771e329d7dc4f1a3ee8fb22acdcb3a75c1c92e75c18R66) is actually an workaround for that issue. Added a comment ao we remove it once 0.59 gets released.

**Review by TylerHelmuth — APPROVED** [review]

File: charts/opentelemetry-operator/values.yaml (hunk 2 above)
Comment (avadhut123pisal) [reply charts/opentelemetry-operator/values.yaml:19]: @Allex1  @TylerHelmuth  As v0.59.0 has been released for the operator, should we modify this PR to use the same ?

Comment (Allex1) [reply charts/opentelemetry-operator/values.yaml]: @avadhut123pisal Done

**Review by dmitryax — APPROVED** [review]

**Pull request merged**